intel_test/host/RunDDRExplicitTest.cpp

```diff
@@ -0,0 +1,56 @@
+#include "../../include/hlslib/intel/OpenCL.h"
+#include <algorithm>
+#include <assert.h>
+#include <iostream>
+
+#define DATA_SIZE 1024
+
+int main(int argc, char **argv) {
+  std::cout << "Initializing OpenCL context..." << std::endl;
+  //"xilinx_u280_xdma_201920_3"
+  hlslib::ocl::Context context;
+  std::cout << "Done." << std::endl << std::flush;
+
+  // Handle input arguments
+  std::string kUsage = "./RunDDRExplicit [emulation|hardware]";
+  if (argc != 2) {
+    std::cout << kUsage << std::flush;
+    return 1;
+  }
+  std::string mode_str(argv[1]);
+  std::string kernel_path;
+  if (mode_str == "emulation") {
+    kernel_path = "DDRExplicit_hw_emu.xclbin";
+  } else if (mode_str == "hardware") {
+    kernel_path = "DDRExplicit_hw.xclbin";
+  } else {
+    std::cout << kUsage << std::flush;
+    return 2;
+  }
+
+  std::cout << std::endl << "Loading Kernel" << std::endl << std::flush;
+  auto program = context.MakeProgram(kernel_path);
+
+  std::cout << "Done" << std::endl
+            << "Initializing memory..." << std::endl
+            << std::flush;
+  std::vector<int, hlslib::ocl::AlignedAllocator<int, 4096>> ddr0mem(DATA_SIZE);
+  std::vector<int, hlslib::ocl::AlignedAllocator<int, 4096>> ddr1mem(DATA_SIZE);
+  std::fill(ddr1mem.begin(), ddr1mem.end(), 15);
+
+  auto memDevice1 = context.MakeBuffer<int, hlslib::ocl::Access::readWrite>(hlslib::ocl::MemoryBank::bank0, DATA_SIZE);
+  auto memDevice2 = context.MakeBuffer<int,hlslib::ocl::Access::readWrite>(hlslib::ocl::MemoryBank::bank1, ddr1mem.begin(), ddr1mem.end());
+
+  std::cout << "Done" << std::endl << std::flush;
+  std::cout << "Running Kernel" << std::endl << std::flush;
```

Comment:
`std::endl` is the same as `"\n" << std::flush`, so there's no point in doing both `std::endl` and `std::flush`. The reason why I sometimes put a flush is because I want to make sure it's flushed to the terminal before it starts running something, but don't necessarily want a newline. If you want to make sure it's flushed _and_ want to line break, just `std::endl`. If you want to flush, but don't want to break the line, use `std::flush`.